clone-edk2.sh

```diff
@@ -3,17 +3,23 @@
 echo "EDK2_VERSION:<$EDK2_VERSION>"
 case "$EDK2_VERSION" in
 	latest)
-		git clone "https://github.com/tianocore/edk2" edk2
+		git clone --depth=1 "https://github.com/tianocore/edk2" edk2
+		;;
+	vUDK2018)
+		git clone -b UDK2018 "https://github.com/tianocore/edk2" edk2
 		;;
 	RefindPlusUDK)
-		git clone "https://github.com/dakanji/RefindPlusUDK" edk2
-		;;	
+		git clone --depth=1 "https://github.com/dakanji/RefindPlusUDK" edk2
+		;;
+	RefindPlusUDKJVT)
+		git clone --depth=1 "https://github.com/joevt/RefindPlusUDK" edk2
+		;;		
 	AcidantheraAUDK)
-		git clone "https://github.com/acidanthera/audk" edk2
+		git clone --depth=1 "https://github.com/acidanthera/audk" edk2
 		;;
 	*)
 		git clone "https://github.com/tianocore/edk2" edk2
 		git -C edk2 checkout "$EDK2_VERSION"
 		;;
 esac
-git -C edk2 submodule update --init
+git -C edk2 submodule update --init 
```

Comment:
Could you please change it to: https://github.com/xaionaro/edk2-builder-docker/pull/24/commits/0ef980762055f305a11e7dedd1a809f15352e9f7

Reply:
done